### PR TITLE
Add example API for verifying custom verifiable presentations

### PR DESCRIPTION
**Pull request by przydatek**
Add to `vc_util` a submodule `custom` with an example implementation of a helper for verifying a verifiable presentation with a `VerifiedAdult`-credential.

### CODE REVIEW

File: src/vc_util/src/custom.rs [hunk 1]
```diff
@@ -0,0 +1,189 @@
...
+use std::collections::HashMap;
+
+/// Validates the provided presentation `vp_jwt`, checking the following conditions:
+///  - the presentation is a valid presentation of the IC Attribute Sharing flow;
+///    in particular the presentation contains IdAlias and a matching VerifiedAdult credentials
+///  - both credentials are valid wrt. `vc_flow_signers`, `root_pk_raw`, and `current_time_ns`
+///  - `effective_vc_subject` matches the principal to which the presentation applies
+pub fn validate_verified_adult_presentation(
+    vp_jwt: &str,
+    effective_vc_subject: Principal,
```
Comment (frederikrothenberger) [inline src/vc_util/src/custom.rs:20]: What is the effective vc subject?

File: src/vc_util/src/custom.rs [hunk 2]
```diff
@@ -0,0 +1,189 @@
+// Custom utils for handling specific VCs/VPs.
```
Comment (frederikrothenberger) [inline src/vc_util/src/custom.rs:1]:
```suggestion
//! Custom utils for handling specific VCs/VPs.
```

File: src/vc_util/src/custom.rs [hunk 3]
```diff
@@ -0,0 +1,189 @@
...
+
+    fn ii_canister_id() -> Principal {
+        Principal::from_text(II_CANISTER_ID).expect("wrong principal")
+    }
+
+    fn issuer_canister_id() -> Principal {
+        Principal::from_text(ISSUER_CANISTER_ID).expect("wrong principal")
+    }
+
+    fn default_test_vc_flow_signers() -> VcFlowSigners {
```
Comment (frederikrothenberger) [inline src/vc_util/src/custom.rs:101]: There should also be test cases that do not use the correct `FcFlowSigners`.

**Review by przydatek — COMMENTED** [review]
Thanks @frederikrothenberger , PTAL.

File: src/vc_util/src/custom.rs (hunk 1 above)
Comment (frederikrothenberger) [reply src/vc_util/src/custom.rs:20]: Maybe a comment explaining this in more detail might be nice?

**Review by frederikrothenberger — APPROVED** [review]
LGTM, thanks!

File: src/vc_util/src/custom.rs (hunk 1 above)
Comment (przydatek) [reply src/vc_util/src/custom.rs:21]: Done.

**Review by przydatek — COMMENTED** [review]
Thanks!

**Pull request merged**